include/zephyr/drivers/usb_c/usbc_vbus.h

```diff
@@ -61,11 +61,16 @@ static inline bool usbc_vbus_check_level(const struct device *dev, enum tc_vbus_
  *
  * @retval 0 on success
  * @retval -EIO on failure
+ * @retval -ENOTSUP if the driver doesn't implement the request
  */
 static inline int usbc_vbus_measure(const struct device *dev, int *meas)
 {
 	const struct usbc_vbus_driver_api *api = (const struct usbc_vbus_driver_api *)dev->api;
 
+	if (api->measure == NULL) {
+		return -ENOTSUP;
```

Comment:
`ENOTSUP` is for an unsupported value.
```suggestion
		return -ENOSYS;
```
Please replace ENOTSUP with ENOSYS throughout.